lib/cdo/aws/cloudfront.rb

```diff
@@ -0,0 +1,211 @@
+require_relative '../../../deployment'
```

Comment:
Does this config end up getting tested?  I see where we do this for varnish (in cdo-varnish/test/integration/default/minitest/test_default.rb) but not a corresponding test for cloudfront.